src/js/core/methods/ComposeTransaction.js

```diff
@@ -59,12 +62,15 @@ export default class ComposeTransaction extends AbstractMethod {
         const { payload } = message;
         // validate incoming parameters
         validateParams(payload, [
-            { name: 'outputs', type: 'array', obligatory: true, allowEmpty: true },
```

Comment:
Why do we remove this? Seems unrelated to the feature in question.

Reply:
i've found it across it during writing test, composing transaction without defined outputs doesnt make sense and also will throw an error from hd-wallet.
validation error from connect will throw more meaningful message than hd-wallet

Reply:
allright. Thanks for clarification. .)